engine/gamesys/src/gamesys/gamesys.cpp

```diff
@@ -104,6 +106,8 @@ namespace dmGameSystem
         REGISTER_RESOURCE_TYPE("meshc", graphics_context, ResMeshPreload, ResMeshCreate, 0, ResMeshDestroy, ResMeshRecreate);
         REGISTER_RESOURCE_TYPE("modelc", graphics_context, ResModelPreload, ResModelCreate, 0, ResModelDestroy, ResModelRecreate);
         REGISTER_RESOURCE_TYPE("materialc", render_context, ResMaterialPreload, ResMaterialCreate, 0, ResMaterialDestroy, ResMaterialRecreate);
+        REGISTER_RESOURCE_TYPE("compute_programc", render_context, ResComputeProgramPreload, ResComputeProgramCreate, 0, ResComputeProgramDestroy, ResComputeProgramRecreate);
+        REGISTER_RESOURCE_TYPE("computec", graphics_context, ResComputeShaderPreload, ResComputeShaderCreate, 0, ResComputeShaderDestroy, ResComputeShaderRecreate);
```

Comment:
Note for reviewer - what should we call these?

A "compute program" is analogous to a fragment/vertex shader and a "compute" is the linked compute program that contains all the resource bind points and is analogous to a material.

Reply:
ah yes! .cp is good! I'll try to keep everything similar to what we already have with vx/frag programs (even though I'd like to call them shaders since it's common nomenclature, but oh well)